CONTRIBUTING.md

```diff
@@ -1 +1,3 @@
-Please see https://yarnpkg.com/org/contributing/
\ No newline at end of file
+Contributing to Yarn V1: please see https://classic.yarnpkg.com/en/docs/getting-started
+
+Contributing to Yarn V2: please see https://yarnpkg.com/en/org/contributing
```

Comment:
FYI,  that link somehow redirects to the classic website (V1 contributing link). I don’t think that’s expected? Shouldn’t it be https://yarnpkg.com/advanced/contributing?